models/profile.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Profile extends Model {
```

Comment:
I don't think we need this model since this column is located in in users model, you delete it I guess